### PR TITLE
Postmanager/get

### PR DESCRIPTION
**Pull request by hewonjeong**
- add getPostKeys function in PostManager
- add mockUserKey in mock-datas for test

### CODE REVIEW

File: src/server/database-mock-data.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 const initTime = new Date();
 const dayInMs = 1000 * 60 * 60 * 24;
+export const mockUserKey = 'user-8000000000000-uuiduuid-uuid-uuid-uuid-uuiduuiduuid';
 export const mockItems = {
   'item-8523910540005-dd3860f5-b82e-473b-1234-ead0f190b000': {
```
Comment (festinalente91) [inline src/server/database-mock-data.js]: what about this form?

```
[
    {
        key: “”,
        value: {
            name: “”,
            pw: “”
        }
    }
]

```

File: src/server/PostManager.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
+import {fetchPrefix} from '../server/database';
+import {STATE} from '../server/key-utils';
 
 export default class PostManager {
-  constructor() {
-
+  static getPostKeys(postType, userKey, cb) {
+    const prefix = `${postType}-${STATE.ALIVE}-${userKey}`;
```
Comment (ovekyc) [inline src/server/PostManager.js]: How about create new STATE for createdpost and savedpost?

File: src/server/PostManager.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
+import {fetchPrefix} from '../server/database';
+import {STATE} from '../server/key-utils';
 
 export default class PostManager {
-  constructor() {
-
+  static getPostKeys(postType, userKey, cb) {
```
Comment (ovekyc) [inline src/server/PostManager.js]: I think type is better than postType

File: src/test/node/PostManager.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  endTime: '2017-01-01T10:10:10.851Z',
+  key: 'item-8520000000000-mockitem-aaaa-aaaa-1234-mockitemuuid',
+  userKey: 'user-8550000000000-mockuser-zzzz-zzzz-1234-mockuseruuid'
+};
+
+test('get created post keys of a user', t => {
+  const expected = {
+    length: Object.keys(mockCreatedPosts).length
+  };
+  PostManager.getPostKeys(ENTITY.CREATED_POST, mockUserKey, (err, values) => {
```
Comment (ovekyc) [inline src/test/node/PostManager.js]: Where is initMock?

File: src/server/database-mock-data.js (hunk 1 above)
Comment (hewonjeong) [reply src/server/database-mock-data.js]: Yap. if we modify as array, it's easier to access key of mock items in test code.
I will apply in other new branch!

File: src/server/PostManager.js (hunk 3 above)
Comment (hewonjeong) [reply src/server/PostManager.js]: Applied~!

File: src/test/node/PostManager.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import {ENTITY} from '../../server/key-utils';
+import PostManager from '../../server/PostManager';
+import {clearDB, initMock} from '../../server/database';
+
+test('get created post keys of a user', t => {
+  const expected = {
+    length: Object.keys(mockCreatedPosts).length
+  };
+  return new Promise((resolve) => {
+    clearDB().then(initMock).then(resolve);
```
Comment (ovekyc) [inline src/test/node/PostManager.js]: .catch(reject)

File: src/test/node/PostManager.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import {mockUserKey, mockCreatedPosts} from '../../server/database-mock-data';
+import {ENTITY} from '../../server/key-utils';
+import PostManager from '../../server/PostManager';
+import {clearDB, initMock} from '../../server/database';
+
+test('get created post keys of a user', t => {
+  const expected = {
+    length: Object.keys(mockCreatedPosts).length
+  };
+  return new Promise((resolve) => {
```
Comment (ovekyc) [inline src/test/node/PostManager.js]: resolve, reject

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**